docs/release-notes/1.5.0-preview2/release-1.5.0-preview2.md

```diff
@@ -0,0 +1,64 @@
+
+
+## **Enhancements**
+- Added in support for System.DateTime type for the DateTimeTransformer ([#4661](https://github.com/dotnet/machinelearning/pull/4661))
+- Additional changes to ExpressionTransformer ([#4614](https://github.com/dotnet/machinelearning/pull/4614))
+
+## **Test updates**
+- **Code analysis updates**
+  - Update analyzer test library ([#4740](https://github.com/dotnet/machinelearning/pull/4740))
+  - Enable the internal code analyzer for test projects ([#4731](https://github.com/dotnet/machinelearning/pull/4731))
+  - Implement MSML_ExtendBaseTestClass (Test classes should be derived from BaseTestClass) ([#4746](https://github.com/dotnet/machinelearning/pull/4746))
+  - Enable MSML_TypeParamName for the full solution ([#4762](https://github.com/dotnet/machinelearning/pull/4762))
+- **Better logging from tests**
+  - Ensure tests capture the full log ([#4710](https://github.com/dotnet/machinelearning/pull/4710))
+  - Fix failure to capture test failures ([#4716](https://github.com/dotnet/machinelearning/pull/4716))
+  - Collect crash dump upload dump and pdb to artifact ([#4666](https://github.com/dotnet/machinelearning/pull/4666))
+- Enable Conditional Numerical Reproducibility for tests ([#4569](https://github.com/dotnet/machinelearning/pull/4569))
+- Changed all MLContext creation to include a fixed seed ([#4736](https://github.com/dotnet/machinelearning/pull/4736))
+- Fix incorrect SynchronizationContext use in TestSweeper ([#4779](https://github.com/dotnet/machinelearning/pull/4779))
+
+## **Documentation Updates**
+- Update documentation to stop mentioning interfaces that no longer exist ([#4673](https://github.com/dotnet/machinelearning/pull/4673))
+- Roadmap update ([#4704](https://github.com/dotnet/machinelearning/pull/4704))
+- Added release process documentation to README.md ([#4402](https://github.com/dotnet/machinelearning/pull/4402))
+- Fix documentation of SvmLightLoader ([#4616](https://github.com/dotnet/machinelearning/pull/4616))
+- Correct KMeans scoring function doc ([#4705](https://github.com/dotnet/machinelearning/pull/4705))
+- Several typo fixes thanks to [@MaherJendoubi](https://github.com/MaherJendoubi) ([#4627](https://github.com/dotnet/machinelearning/pull/4627), [#4631](https://github.com/dotnet/machinelearning/pull/4631), [#4626](https://github.com/dotnet/machinelearning/pull/4626) [#4617](https://github.com/dotnet/machinelearning/pull/4617), [#4633](https://github.com/dotnet/machinelearning/pull/4633), [#4629](https://github.com/dotnet/machinelearning/pull/4629), [#4642](https://github.com/dotnet/machinelearning/pull/4642))
+- Other typo fixes:([#4628](https://github.com/dotnet/machinelearning/pull/4628), [#4685](https://github.com/dotnet/machinelearning/pull/4685))
+
+## **CLI and AutoML API**
+  - CodeGen For AzureAttach ([#4498](https://github.com/dotnet/machinelearning/pull/4498))
```

Comment:
We don't want user aware of this change because this is only for ModelBuilder Azure Attach story

Reply:
plase add
 - Using invariance culture when converting to string ( [#4635](https://github.com/dotnet/machinelearning/pull/4635) )
and 
 - Fix NullReferenceException when it comes to Recommendation in AutoML and CodeGenerator ( [#4774](https://github.com/dotnet/machinelearning/pull/4774) )

Thanks